ddtrace/internal/_unpatched.py

```diff
@@ -0,0 +1,10 @@
+# Acquire a reference to the open function from the builtins module. This is
+# necessary to ensure that the open function can be used unpatched when required.
+from builtins import open as unpatched_open  # noqa
+
+# Acquire a reference to the threading module. Some parts of the library (e.g.
+# the profiler) might be enabled programmatically and therefore might end up
+# getting a reference to the tracee's threading module. By storing a reference
+# to the threading module used by ddtrace here, we make it easy for those parts
+# to get a reference to the right threading module.
+import threading as _threading  # noqa
```

Comment:
would `import threading` be enough here? we did `as _threading` before to make sure to not expose it as part of our public API

🤔

Reply:
this one makes life easier because I need to update all `from ddtrace import _threading`